zerver/lib/actions.py

```diff
@@ -1678,6 +1678,13 @@ def do_change_full_name(user_profile, full_name, log=True):
         send_event(dict(type='realm_bot', op='update', bot=payload),
                    bot_owner_userids(user_profile))
 
+def do_update_full_name_in_admin_ui(user_profile, target_user, full_name):
+    # type: (UserProfile, UserProfile, text_type) -> None
+    payload = dict(email=target_user.email,
+                   full_name=target_user.full_name)
+    send_event(dict(type='admin_user', op='update_full_name', user=payload),
+               [user_profile.id])
```

Comment:
There's no need for this function or a new event type -- from a client perspective it doesn't matter who changed that user's name.  We should instead just make the admin UI update based on user full name changes.